pom.xml

```diff
@@ -59,13 +59,27 @@
     <dependency>
       <groupId>org.kohsuke</groupId>
       <artifactId>github-api</artifactId>
-      <version>${github-api.version}</version>
+      <version>${github-api.version}-SNAPSHOT</version>
+      <classifier>shaded</classifier>
+      <!-- The shaded jar has all non-optional dependencies included. But this is the base pom, so we need to exclude. -->
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
     <dependency>
       <groupId>com.squareup.okhttp</groupId>
       <artifactId>okhttp-urlconnection</artifactId>
       <version>2.7.5</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.10.2</version>
+      <scope>test</scope>
```

Comment:
Even though it's only in test scope, I think it would make more sense to use the [jackson2-api](https://github.com/jenkinsci/jackson2-api-plugin) plugin, just to avoid any issues in case of future POM refactoring.